core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -184,7 +184,10 @@ object Boot extends App with LazyLogging {
       gcsConfig.getInt("groupMonitor.workerCount"),
       userServiceConstructor))
 
-    BootMonitors.restartMonitors(slickDataSource, gcsDAO, submissionSupervisor, bucketDeletionMonitor)
+    if(sys.env.getOrElse("BACK_RAWLS", "false").toBoolean) {
+      logger.info("This instance has been marked as BACK_RAWLS. Booting monitors...")
+      BootMonitors.restartMonitors(slickDataSource, gcsDAO, submissionSupervisor, bucketDeletionMonitor)
```

Comment:
quirk: we still create the supervisor for submission monitoring but it will be unused for front-rawls. 

it felt cleaner and less quirky than having the supervisor missing entirely

Reply:
I am not sure I agree... why is it cleaner and less quirky?

Reply:
i think it's cleaner because there's less logic separating the two types of rawls. every rawls ends up being constructed the same, it's just a matter of flipping the switch on the monitors or not